src/code/z_DLF.c

```diff
@@ -2,6 +2,74 @@
 #include "system_malloc.h"
 #include "z64load.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_DLF/Overlay_LoadGameState.s")
+void Overlay_LoadGameState(GameStateOverlay* gameState) {
```

Comment:
I think I would prefer to name it to `gameStateOvl` or similar. Just plain `gameState` makes me think it is related to the GameState struct